include/ddc/kernels/splines/spline_builder_batched.hpp

```diff
@@ -127,6 +127,24 @@ class SplineBuilderBatched
         return spline_tr_domain_type(bsplines_domain(), batch_domain());
     }
 
+    derivs_domain_type const derivs_xmin_domain() const noexcept
+    {
+        return ddc::replace_dim_of<interpolation_mesh_type, deriv_type>(
+                vals_domain(),
+                ddc::DiscreteDomain<deriv_type>(
+                        ddc::DiscreteElement<deriv_type>(1),
+                        ddc::DiscreteVector<deriv_type>(std::max(1, s_nbc_xmin))));
```

Comment:
std::max is used because I guess we cannot build a DiscreteDomain of size 0. User has to handle himself this singular case.

Reply:
Size 0 should be handled, did you try ?